elastic/src/main/java/org/openmbee/mms/elastic/CommitElasticDAOImpl.java

```diff
@@ -21,11 +18,15 @@
 import org.openmbee.mms.elastic.utils.Index;
 import org.openmbee.mms.json.BaseJson;
 import org.openmbee.mms.json.CommitJson;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Component;
 
 @Component
 public class CommitElasticDAOImpl extends BaseElasticDAOImpl<CommitJson> implements CommitIndexDAO {
 
+    @Value("${elasticsearch.limit.index}")
+    int indexLimit;
```

Comment:
should we be using the same limit for indexing vs splitting up commit? the commit can be a lot bigger than whatever we set the index limit to (currently used in bulk processor) (this is already injected in the BaseDao as bulkLimit)